exercises/knapsack/canonical-data.json

```diff
@@ -0,0 +1,20 @@
+{
+  "exercise": "knapsack",
+  "version": "1.0.0",
+  "comments": [
+    "Depending on the language, the input type can be modified"
+  ],
+  "cases": [
+    {
+      "description": "simple knapsack",
+      "property": "maximum value",
+      "input": {
+        "W": 10,
+        "N": 4,
+        "w": [10, 40, 30, 50],
+        "v": [5, 4, 6, 3]
+      },
+      "expected": 90
+    },
```

Comment:
Trailing comma looks wrong though will be needed if you are planning to add more tests.  I think presently Travis CI is failing because of it.

Reply:
Yup fixed tests!